cmd/keymaster/main.go

```diff
@@ -411,15 +464,17 @@ func main() {
 	if *cliUsername != "" {
 		userName = *cliUsername
 	}
-
 	if len(config.Base.FilePrefix) > 0 {
 		FilePrefix = config.Base.FilePrefix
 	}
 	if *cliFilePrefix != "" {
 		FilePrefix = *cliFilePrefix
 	}
-
-	err = setupCerts(userName, homeDir, config, client, logger)
+	if flag.Arg(0) == "aws-role-cert" {
+		err = generateAwsRoleCert(homeDir, config, client, logger)
+	} else {
+		err = setupCerts(userName, homeDir, config, client, logger)
+	}
```

Comment:
This is soo ugly... but I cannot think of a better way unless we write more tooling....